siliconcompiler/core.py

```diff
@@ -2791,17 +2791,17 @@ def _generate_summary_image(self, input_path, output_path):
         # "final" metrics regardless of flow would be handy
         totalarea = self.get('metric', 'totalarea', step='export', index='1')
         if totalarea:
-            metrics['Area'] = f'{int(totalarea)} um^2'
+            metrics['Area'] = f'{totalarea:.2f} um^2'
```

Comment:
I meant to ask, should we scale this to mm^2 when it gets too big?

Reply:
yes

Reply:
I'll leave the trigger point up to you, in the past I've usually gone a little past the SI breakpoint so 10e6 um2 -> 10.00mm^2, instead of 1e6, but I'm not wed to either